src/subnet.tf

```diff
@@ -10,13 +10,4 @@ resource "azurerm_subnet" "main" {
   virtual_network_name = local.vnet_name
   address_prefixes     = [var.database.cidr]
   service_endpoints    = ["Microsoft.AzureCosmosDB"]
-  delegation {
```

Comment:
It feels like we'd want to keep this even if we get rid of the `cmk`. Can you explain why we are removing this now?

Reply:
I discovered we don't need it, I just added it because PSQL and MySQL bundles do need it.

Reply:
Okay got it, thank you